FundParser/BL/Services/FundCsvService/FundCsvService.cs

```diff
@@ -50,15 +50,14 @@ public async Task<int> UpdateHoldings(CancellationToken cancellationToken = defa
         {
             await _logger.LogError($"{CsvUrlSectionKey} is not set in the configuration",
                 nameof(FundCsvService), cancellationToken);
-            throw new Exception($"{CsvUrlSectionKey} is not set in the configuration");
+            throw new MissingConfigurationException($"{CsvUrlSectionKey} is not set in the configuration");
+        }
+        var csvString = await _downloaderService.DownloadTextFileAsStringAsync(url, _csvRequestHeaders, cancellationToken);
+        var csvRows = _csvParsingService.ParseString(csvString, cancellationToken);
+        if (!csvRows.Any())
+        {
+            return -1;
```

Comment:
I think we should either return `0` here, or adjust the condition in `CsvContoller` to `<= 0)`, currently, it is:
```
        if (result == 0)
        {
            return Ok("No rows were updated!");
        }
```